orcid-integration-test/src/test/java/org/orcid/integration/blackbox/api/v2/rc1/VerifyOrcidBeforeFetchElementTest.java

```diff
@@ -38,7 +38,7 @@
 import org.orcid.api.common.WebDriverHelper;
 import org.orcid.integration.api.pub.PublicV2ApiClientImpl;
 import org.orcid.jaxb.model.common_rc1.Visibility;
-import org.orcid.jaxb.model.error.OrcidError;
+import org.orcid.jaxb.model.error_rc2.OrcidError;
```

Comment:
yes @nitw-shobhit look at the package name: 

blackbox/api/v2/rc1/VerifyOrcidBeforeFetchElementTest.java

..../v2/rc1/....